distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -242,16 +242,19 @@ type File
     resolve : (Text | File) -> File
     resolve self = @Builtin_Method "File.resolve"
 
-    ## A File to JSON conversion.
+    ## Convert to a JavaScript Object representing this File.
 
        > Example
-         Get a JSON representation of a file.
+         Convert a file to a JS_Object.
```

Comment:
```suggestion
         Convert the file descriptor to a JS_Object.
```

(the wording can be different) But the current wording could be understood as converting the _file_ i.e. its **contents** to JSON. But that is not the case. I suggest rephrasing to minimize user confusion.